data/json/furniture.json

```diff
@@ -993,6 +993,42 @@
         }
     },{
         "type" : "furniture",
+        "id" : "f_bigmirror",
+        "name": "standing mirror",
+        "symbol": "{",
+        "color": "white",
+        "move_cost_mod": 2,
+        "required_str": 5,
+        "flags": ["NOITEM","BLOCKSDOOR"],
+        "bash": {
+            "str_min": 5, "str_max": 40,
```

Comment:
EDIT: I meant this one, not the broken one. With str_max 40 it takes a few hits to break.